lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
Rules having a length of 0 is a valid case.
Example: `tldjs.rules.eu.length` => 0

"Nothing found" would mean the tld is undefined, so the check against void 0 should be the correct one.
Example: `tldjs.rules.imagination === void 0` => true